packages/fluentui/react-northstar/src/components/Carousel/CarouselNavigation.tsx

```diff
@@ -59,6 +59,9 @@ export interface CarouselNavigationProps extends UIComponentProps, ChildrenCompo
 
   /** A vertical carousel navigation displays elements vertically. */
   vertical?: boolean;
+
+  /** A navigation may be clickable */
+  disableClickableNav?: boolean;
```

Comment:
Could you provide more details in the PR description ? Do you mean these dots ?

![image](https://user-images.githubusercontent.com/20744592/125687544-210de753-f409-448a-9487-2d8d0e2a49e3.png)

Reply:
Yes, I want to make this dots can be unclickable.

Reply:
I don't quite understand then, those dots are already unclickable for me in the docsite examples ?

Reply:
@ling1726 yes, this dots are already unclickable in the docs, but our team find that sometimes this will be clickable, per offline discussion with Juraj, there might be some regression and he suggested us to contribute to Fluent UI to allow the dots to be unclicable. Thanks